webapp/channels/src/components/confirm_modal.tsx

```diff
@@ -166,46 +167,52 @@ export default class ConfirmModal extends React.Component<Props, State> {
             );
         }
 
+        const headerText = (
+            <div className='ConfirmModal__header'>
+                <span id='confirmModalLabel'>
+                    {this.props.title}
+                </span>
+            </div>
+        );
+
         return (
-            <Modal
+            <GenericModal
                 id={classNames('confirmModal', this.props.id)}
-                className={'modal-confirm ' + this.props.modalClass}
-                dialogClassName='a11y__modal'
+                className={'modal-confirm ' + this.props.modalClass + ' a11y__modal ConfirmModal'}
```

Comment:
Could the references to `modal-confirm` and `ConfirmModal` be consolidated? It seems odd to have two nearly identical CSS classes on this

Reply:
I agree that having both ‘modal-confirm’ and ‘ConfirmModal’ is redundant. I’ll consolidate these classes into a single, consistent naming scheme to keep our styles simpler and more maintainable